apps/amo/tests/test_csp_headers.py

```diff
@@ -0,0 +1,95 @@
+from django.conf import settings
+
+import lib.settings_base as base_settings
+
+import amo.tests
+import mock
+
+
+@mock.patch.object(settings, 'CSP_REPORT_ONLY', False)
+class TestCSPHeaders(amo.tests.TestCase):
+
+    def test_for_specific_csp_settings(self):
+        """Test that require settings are provided as headers."""
+        response = self.client.get('/en-US/firefox/')
+        assert response.status_code == 200
+        # Make sure a default-src is locked down.
+        assert "default-src 'none'" in response['content-security-policy']
+        # Make sure a object-src is locked down.
+        assert "object-src 'none'" in response['content-security-policy']
+        # The report-uri should be set.
+        assert "report-uri" in response['content-security-policy']
+        # Basic assertions on the things we've defined.
+        assert "script-src" in response['content-security-policy']
+        assert "style-src" in response['content-security-policy']
+        assert "font-src" in response['content-security-policy']
+        assert "frame-src" in response['content-security-policy']
+
+    def test_unsafe_inline_not_in_script_src(self):
+        """Make sure a script-src does not have unsafe-inline."""
+        assert "'unsafe-inline'" not in base_settings.CSP_SCRIPT_SRC
```

Comment:
I would, but then I'd need to start parsing the header. As django-csp has tests and this is more direct it seemed like a better bet.